src/plugins/mer/merdeploysteps.cpp

```diff
@@ -123,6 +123,26 @@ class MerPrepareTargetStepConfigWidget : public BuildStepConfigWidget
     }
 };
 
+class ElidedWebsiteLabel : public QLabel
+{
+public:
+    explicit ElidedWebsiteLabel(const QString &text, QWidget *parent=nullptr, Qt::WindowFlags f=Qt::WindowFlags()) :
+        QLabel("", parent, f)
+    {
+        m_text = text;
+    }
+    void resizeEvent(QResizeEvent *event) {
+        QWidget::resizeEvent(event);
+
+        QFontMetrics metrics(font());
+        QString elidedText = metrics.elidedText(m_text, Qt::ElideRight, width());
+        setText(QString("<a href='%1'>%2</a>").arg(m_text, elidedText));
```

Comment:
Tricky. If the font used for links differs...

Reply:
and so we put the setOpenExternalLinks outside

Reply:
Let's assume this won't happen